adapters/nextmillennium/nextmillennium.go

```diff
@@ -98,12 +86,18 @@ func (adapter *adapter) buildAdapterRequest(prebidBidRequest *openrtb2.BidReques
 		Headers: headers}, nil
 }
 
-func createBidRequest(prebidBidRequest *openrtb2.BidRequest, params *openrtb_ext.ImpExtNextMillennium) *NextMillenniumBidRequest {
+type nmExtPrebidStoredRequest struct {
+	ID string `json:"id"`
+}
+type nmExtPrebid struct {
+	Storedrequest nmExtPrebidStoredRequest `json:"storedrequest"`
```

Comment:
Please follow the general naming convention and rename `Storedrequest` to `StoredRequest`

Reply:
I changed it, 
Thanks!